source/jormungandr/jormungandr/instance.py

```diff
@@ -748,8 +748,15 @@ def _send_and_receive(
         logger = logging.getLogger(__name__)
         deadline = datetime.utcnow() + timedelta(milliseconds=timeout)
         request.deadline = deadline.strftime('%Y%m%dT%H%M%S,%f')
-        pt_socket = request.requested_api == type_pb2.pt_planner
-        with self.socket(self.context, pt_socket) as socket:
+        # we use the pt_socket if :
```

Comment:
it think it would be a little bit clearer you pass this parameter where the pt_journey is requested... so don't have to check here

Reply:
Allright, done